docs/UserGuide/Operation Manual/DDL Data Definition Language.md

```diff
@@ -309,6 +310,20 @@ IoTDB> unset ttl to root.ln
 
 After unset TTL, all data will be accepted in `root.ln`
 
+## Show TTL
```

Comment:
same as neuyilan's suggestion~

Reply:
Hi, I changed the Chinese document in the PR submitted earlier,

Reply:
> same as neuyilan's suggestion~

Hi, I changed the Chinese document in the PR submitted earlier,